docs/api_scrubadub_detectors.rst

```diff
@@ -10,140 +10,237 @@ text.
 Base classes
 ------------
 
-Every ``Detector`` that inherits from ``scrubadub.detectors.base.Detector``.
+Every ``Detector`` that inherits from ``scrubadub.detectors.Detector``.
 
-scrubadub.detectors.base.Detector
----------------------------------
+.. _scrubadub.detectors.Detector:
```

Comment:
I showed this to you earlier - the dict didn't render well 

`__dict__= mappingproxy({'__module__': 'scrubadub.detectors.base', '__doc__':`